src/summariser.py

```diff
@@ -2,17 +2,48 @@
 from cohere.types.non_streamed_chat_response import NonStreamedChatResponse
 from src.utils import get_cohere_api_key
 import logging
+from langchain_community.document_loaders import WebBaseLoader
+import re
+from langchain_core.documents.base import Document
 
 logger = logging.getLogger(__name__)
 
 
+def extract_urls(text):
```

Comment:
Separate thread on the same line, docstrings are also really nice. I have a VSCode extension that automatically helps with these: `njpwerner.autodocstring`

A good docstring has a general description, typed arguments list with parameter descriptions for each and example usage, any exceptions your function raises and why it would raise them, and return value explanation

Reply:
[added](https://github.com/tommylees112/whatsappLLM/pull/2/commits/d9e3673d0c48d30dcd35adb2fcb5f012d2d04471).